src/main/java/org/ecocean/servlet/importer/StandardImport.java

```diff
@@ -894,10 +931,40 @@ public Encounter loadEncounter(Row row, ArrayList<Annotation> annotations, Strin
     if (sex!=null) enc.setSex(sex);
 
     String genus = getString(row, "Encounter.genus",colIndexMap, verbose, missingColumns, unusedColumns, feedback);
-    if (genus!=null) enc.setGenus(genus);
+    boolean hasGenus=false;
+    if (genus!=null && !genus.trim().equals("")) {
+    	hasGenus=true;
+    	enc.setGenus(genus.trim());
+    }
 
     String specificEpithet = getString(row, "Encounter.specificEpithet",colIndexMap, verbose, missingColumns, unusedColumns, feedback);
-    if (specificEpithet!=null) enc.setSpecificEpithet(specificEpithet);
+    boolean hasSpecificEpithet=false;
+    if (specificEpithet!=null && !specificEpithet.trim().equals("")) {
+    	hasSpecificEpithet=true;
+    	enc.setSpecificEpithet(specificEpithet.trim());
+    }
+    
+    //start check for missing or unconfigured genus+species
+    if (!hasGenus) {
+        //mark genus empty
+    	feedback.logParseError(getColIndexFromColName("Encounter.genus", colIndexMap),"GENUS", row, "MISSING GENUS");
+    }
+    if (!hasSpecificEpithet) {
+        //mark specific epithet
+        feedback.logParseError(getColIndexFromColName("Encounter.specificEpithet", colIndexMap),"SPECIFIC EPITHET", row, "MISSING SPECIFIC EPITHET");
+    }
+    //now validate that a present genus and species value are supported
+    if(hasGenus && hasSpecificEpithet) {
+    	
+    	List<String> configuredSpecies = CommonConfiguration.getIndexedPropertyValues("genusSpecies", myShepherd.getContext());
```

Comment:
how is this _not_ already a method we have somewhere? :exploding_head: (i looked, and cant seem to find it [either]. closest might be `Shepherd.getAllTaxonomyNames()`.) i wonder if we should make this a method we can use in a hopeful future - maybe inside Taxonomy class?

Reply:
i meant the above line _plus_ the one below: basically **validating** a genus/specificEpithet pair.

Reply:
Good idea. I will implement this in the next pass at adding checks.